newIDE/app/src/UI/Card.js

```diff
@@ -38,9 +39,33 @@ const Card = ({
   cardCornerAction,
   isHighlighted,
   disabled,
+  padding,
 }: Props) => {
   const { isMobile } = useResponsiveWindowSize();
   const gdevelopTheme = React.useContext(GDevelopThemeContext);
+  const cardContentStyle: {|
+    minWidth: number,
+    padding?: number | string,
+    marginLeft?: number,
+    paddingTop?: number,
+    paddingBottom?: number,
+    paddingLeft?: number,
+    paddingRight?: number,
+  |} = {
+    ...styles.cardContent,
+    ...(padding !== undefined
+      ? { padding }
+      : {
+          paddingTop: 0,
+          paddingBottom: 18,
+          paddingRight: isMobile ? 8 : 24,
+          paddingLeft: isMobile ? 8 : 24,
+        }),
+  };
+  if (isHighlighted) {
+    // Compensate offset brought by left border
+    cardContentStyle.marginLeft = -highlightedBorderWidth;
```

Comment:
I'd be careful with minus margins like this, on mobile it creates an unnecessary horizontal scroll most of the time. 
Are you sure it's ok?

Reply:
I see no issue on mobile but it's a good reflex I guess to be scared 😅  I'll see if I can fix the padding